transform/mattermost-analytics/models/staging/geolite/_geolite__models.yml

```diff
@@ -27,6 +27,25 @@ models:
     columns:
       - name: network
         description: The IPv4 network in CIDR format such as "2.21.92.0/29".
+      - name: cidr_prefix
```

Comment:
based on this: https://docs.digitalocean.com/glossary/cidr/ - sounds like this is the CIDR prefix length.

Reply:
Renaming to `cidr_network_size`